src/store/modules/cart.ts

```diff
@@ -9,13 +9,13 @@ const getters = {
     cartItems(state: CartState): CartItem[] {
         return state.items;
     },
-    priceTotal(state: CartState) {
+    priceTotal(state: CartState) :number {
         let priceTotal = 0;
         for (const item of state.items) {
             priceTotal += (item.price * item.quantity);
         }
         return state.priceTotal = priceTotal;
```

Comment:
you're returning this value or adding it to state? seems return is unnecessary and functions should be void